crates/hyperswitch_connectors/src/connectors/digitalvirgo/test.rs

```diff
@@ -0,0 +1,402 @@
+use masking::Secret;
+use router::{
+    types::{self, api, storage::enums,
+}};
+
+use crate::utils::{self, ConnectorActions};
```

Comment:
Please rename this file to digitalvirgo.rs and move it to `crates/router/tests/connectors`